6/README.md

```diff
@@ -0,0 +1,1078 @@
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Get pipe list for a given Tango class with a specified filter </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetDevicePipeList.html"> DbGetDevicePipeList </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Return list of pipes matching the wildcard for the specified device </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbDeleteAllDevicePipeProperty.html"> DbDeleteAllDevicePipeProperty </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEV_VOID </td>
+		<td> OPERATOR </td>
+		<td> Delete all pipe properties for the specified device pipe(s) </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbPutClassPipeProperty.html"> DbPutClassPipeProperty </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEV_VOID </td>
+		<td> OPERATOR </td>
+		<td> Create/Update class pipe property(ies) in database </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbPutDevicePipeProperty.html"> DbPutDevicePipeProperty </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEV_VOID </td>
+		<td> OPERATOR </td>
+		<td> Create/Update device pipe property(ies) in database </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetClassPipePropertyHist.html"> DbGetClassPipePropertyHist </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Retrieve Tango class pipe property history </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetDevicePipePropertyHist.html"> DbGetDevicePipePropertyHist </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Retrieve device pipe property history </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetForwardedAttributeListForDevice.html"> DbGetForwardedAttributeListForDevice </a> </td>
+		<td> DEV_STRING </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Get the list of devices using forwarded attribute(s) from specified device </td>
+	</tr>
+</table>
+
+## Database as a file
+For Device servers not able to access the Tango database (most of the time due to network route or security reason), 
+it is possible to start them using a file instead of a real database. This is done via the Device server command line option:  
+```-file=<file name>```  
+
+In this case, the Database SHALL handle the following functionality using the specified file:  
+* getting, setting and deleting class properties  
+* getting, setting and deleting device properties  
+* getting, setting and deleting class attribute properties  
+* getting, setting and deleting device attribute properties  
+
+The following set of limitations exist:  
+* no check that the same Device server is running twice
+* no Device or attribute alias name
+* in case of several Device servers running on the same host, the user must manually manage a list of already used network ports
+
+### File syntax
+The file is an ASCII file and SHALL follow a well-defined syntax with predefined keywords.
```

Comment:
The file format is defined in properties RFC.  Shall we move it to this RFC?

Reply:
I am not seeing the file format in this PR https://github.com/bourtemb/rfc/tree/properties-rfc-5-2nd-try/5.  Maybe I am looking at the wrong place.

I will leave the File Syntax in the Database RFC just for now, until it is decided.

Reply:
Indeed, I removed it from RFC-5 when I rewrote it because this was probably not the best location to put it.
The file syntax was described very briefly in the previous version:
https://github.com/tango-controls/rfc/pull/30/files#diff-192b10cf557bb491182f248eeaf39ae2R181-R188

A very simple example was provided. The syntax was not described using ABNF, which could be a more rigorous way to describe it.